components/event-publisher-proxy/pkg/handler/handler_test.go

```diff
@@ -418,15 +425,20 @@ func TestHandler_publishLegacyEventsAsCE(t *testing.T) {
 					},
 					BackendURL: "FOO",
 				},
-				LegacyTransformer: legacy.NewTransformer("namespace", "im.a.prefix", NewApplicationListerOrDie(context.Background(), "testapp")),
-				collector:         metrics.NewCollector(latency),
-				eventTypeCleaner:  eventtypetest.CleanerStub{},
+				LegacyTransformer: legacy.NewTransformer(
+					"namespace",
+					"im.a.prefix",
+					NewApplicationListerOrDie(context.Background(), "testapp")),
+				collector:        metrics.NewCollector(latency),
+				eventTypeCleaner: eventtypetest.CleanerStub{},
 			},
 			args: args{
 				request: legacytest.ValidLegacyRequestOrDie(t, "v1", "testapp", "object.created"),
 			},
 			wantStatus: 200,
 			wantOk:     true,
+
```

Comment:
with so many nolint directives, you could just apply the `nolint:lll` for the whole file in the first line of this file. That will save some line of code. The drawback of this of course, is that it will be applied for all the code including the function signatures.

Reply:
sounds like a valid solution 👍